apps/src/music/player/soundSub.js

```diff
@@ -85,12 +84,18 @@ WebAudio.prototype.LoadSound = function (url, callback, onLoadFinished) {
           onLoadFinished();
         },
         function (e) {
-          Lab2MetricsReporter.logError('Error decoding audio data', e);
+          Lab2MetricsReporter.logError(
```

Comment:
nit: Looks like these 2 `Lab2MetricsReporter.logError` calls are identical. Can be declared as a named function like `const handleError = e => Lab2MetricsReporter.logError(....` so you won't need to write 2 same blocks of code in the future I guess?